.evergreen/auth_aws/setup_secrets.py

```diff
@@ -0,0 +1,68 @@
+import json
+import os
+import sys
+import yaml
+
+import boto3
+
+HERE = os.path.abspath(os.path.dirname(__file__))
+aws_lib = os.path.join(os.path.dirname(HERE), 'auth_aws', 'lib')
+sys.path.insert(0, aws_lib)
+
+DEFAULT_CLIENT = "0oadp0hpl7q3UIehP297"
+
+
+def get_secrets(profile, *vaults):
+    """Get the driver secret values."""
+    # Handle local credentials.
+    if len(profile) != 0:
```

Comment:
`profile` can have an optional name of "default", since that is what the `Session` object has as a default.